gemini/__init__.py

```diff
@@ -1,8 +0,0 @@
-import os
```

Comment:
Indeed, and it's actually imported just below that function. I suppose the purpose was backwards compatibility?